media/CfgEditor/cfgeditor.js

```diff
@@ -0,0 +1,325 @@
+  codegenEnabled.style.display = 'block';
+  codegenDisabled.style.display = 'none';
+  panelCodegenEnabled.style.display = 'block';
+  panelCodegenDisabled.style.display = 'none';
+
+  clickUnfoldAdvencedOptions();
+  const advancedOptions = document.getElementById('advancedOptions');
+  advancedOptions.activeid = 'tabCodegen';
+}
+
+function disableCodegenStep() {
+  const codegenEnabled = document.getElementById('codegenEnabled');
+  const codegenDisabled = document.getElementById('codegenDisabled');
+  const panelCodegenEnabled = document.getElementById('panelCodegenEnabled');
+  const panelCodegenDisabled = document.getElementById('panelCodegenDisabled');
+
+  codegenEnabled.style.display = 'none';
+  codegenDisabled.style.display = 'block';
+  panelCodegenEnabled.style.display = 'none';
+  panelCodegenDisabled.style.display = 'block';
+}
+
+function enableProfileStep() {
+  const profileEnabled = document.getElementById('profileEnabled');
+  const profileDisabled = document.getElementById('profileDisabled');
+  const panelProfileEnabled = document.getElementById('panelProfileEnabled');
+  const panelProfileDisabled = document.getElementById('panelProfileDisabled');
+
+  profileEnabled.style.display = 'block';
+  profileDisabled.style.display = 'none';
+  panelProfileEnabled.style.display = 'block';
+  panelProfileDisabled.style.display = 'none';
+
+  clickUnfoldAdvencedOptions();
+  const advancedOptions = document.getElementById('advancedOptions');
+  advancedOptions.activeid = 'tabProfile';
+}
+
+function disableProfileStep() {
+  const profileEnabled = document.getElementById('profileEnabled');
+  const profileDisabled = document.getElementById('profileDisabled');
+  const panelProfileEnabled = document.getElementById('panelProfileEnabled');
+  const panelProfileDisabled = document.getElementById('panelProfileDisabled');
+
+  profileEnabled.style.display = 'none';
+  profileDisabled.style.display = 'block';
+  panelProfileEnabled.style.display = 'none';
+  panelProfileDisabled.style.display = 'block';
+}
+
+function modelTypeClick() {
+  const modelType = document.getElementById('modelTypeRadio').value;
+  const InitialImportState = document.getElementById('ImportInitialState');
+  const TFConverterOptions = document.getElementById('TFConverterOptions');
+  const TFLConverterOptions = document.getElementById('TFLITEConverterOptions');
+  const ONNXConverterOptions = document.getElementById('ONNXConverterOptions');
+
+  InitialImportState.style.display = 'none';
+  TFConverterOptions.style.display = 'none';
+  TFLConverterOptions.style.display = 'none';
+  ONNXConverterOptions.style.display = 'none';
+
+  switch (modelType) {
+    case 'pb':
+    case 'savedModel':
+    case 'kerasModel':
+      TFConverterOptions.style.display = 'block';
+      return;
+    case 'tflite':
+      TFLConverterOptions.style.display = 'block';
+      return;
+    case 'onnx':
+      ONNXConverterOptions.style.display = 'block';
+    default:
+      return;
```

Comment:
Just for curious. Not adding `return` in `case onnx:` is a preference, or any software engineering something..?

Reply:
Not preference, not engineering, but my mistake.... T.T
Thanks!!